src/components/main/mainsection-blocks/memberSec/memberMainSec.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {
+    FaRegCheckCircle,
+    FaPlusCircle,
+    FaRegArrowAltCircleUp,
+    FaMapMarkerAlt,
+} from 'react-icons/fa';
+import paul from '../../../../images/main-section/paul.png';
+import victoria from '../../../../images/main-section/victoria.png';
+import joseph from '../../../../images/main-section/joseph.png';
+import MemberRowRows from './memberRowRows';
+
+const MemberMainSec = () => {
+    const member = [
+        {
+            id: 'ac-1',
+            memberFoto: <img className="member-photo" src={paul} alt="paul" />,
+            memberName: 'Paul Robert Smit',
+            memberLoc: ' Manhathan, United State',
+            memberIconDone: <FaRegCheckCircle size="24px" color="#00ff37" />,
+            memberIcon: <FaPlusCircle size="24px" />,
+        },
+        {
+            id: 'ac-2',
+            memberFoto: (
+                <img className="member-photo" src={victoria} alt="victoria" />
+            ),
+            memberName: 'Victoria Campel',
+            memberLoc: 'Barcelona, Spain',
+            memberIconDone: <FaRegCheckCircle size="24px" color="#00ff37" />,
+            memberIcon: <FaRegArrowAltCircleUp size="24px" />,
+            memberNone: <div className="members-main-block-section-row-none" />,
+            memberRows: <MemberRowRows />,
+        },
+        {
+            id: 'ac-3',
+            memberFoto: <img className="member-photo" src={joseph} alt="joseph" />,
```

Comment:
why do you need to put a heavy img tag here. it can be rendered in map. you are repeating same tag inside array all teh times. use just attributes for the img here